### DISCUSSION TITLE
Nemo: Relative Symlinks

### DISCUSSION THREAD

**Discussion by trymeouteh** [linuxmint/discussions#414] [Ideas]
Please add the ability to right click on a file or folder and click copy or by selecting the file or folder and pressing CTRL+C and then right clicking in another directory and select the options "Create Relative Symbolic Link" which will create a relative symlink of the item you have copied.

Will be much easier creating relative symlinks this way then having to use the terminal.

https://github.com/linuxmint/nemo/issues/2784

**Comment by Jeremy7701** [comment]
If you do this often why not write a little BASH script to do it?
It seems unlikely that nemo are going to make it a feature, since "standard" symbolic links are already provided.

Put your script in ~/.local/share/nemo/scripts/
Use F3 to select dual panes - one pane for the link directory where the relative link is to be created and one pane for where it should point to.
The following environment variables are provided by nemo:-

```
In all cases, the following environment variables will be
set by Nemo, which the scripts may use:

- NEMO_SCRIPT_SELECTED_FILE_PATHS: newline-delimited paths for selected files (only if local)
- NEMO_SCRIPT_SELECTED_URIS: newline-delimited URIs for selected files
- NEMO_SCRIPT_CURRENT_URI: URI for current location
- NEMO_SCRIPT_WINDOW_GEOMETRY: position and size of current window
- NEMO_SCRIPT_NEXT_PANE_SELECTED_FILE_PATHS: newline-delimited paths for selected files in the inactive pane of a split-view window (only if local)
- NEMO_SCRIPT_NEXT_PANE_SELECTED_URIS: newline-delimited URIs for selected files in the inactive pane of a split-view window
- NEMO_SCRIPT_NEXT_PANE_CURRENT_URI: URI for current location in the inactive pane of a split-view window
- ```
